lemur/plugins/lemur_sftp/plugin.py

```diff
@@ -113,7 +113,7 @@ def open_sftp_connection(self, options):
             ssh = paramiko.SSHClient()
 
             # allow connection to the new unknown host
-            ssh.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+            ssh.set_missing_host_key_policy(paramiko.RejectPolicy())
```

Comment:
@dmitryzykov as the original plugin author, do you have any input on whether `AutoAddPolicy` was really required?

Reply:
tagging @peschmae to also weigh in, since he has contributed to this plugin, and use it as part of the http01 flow. 
if no objections raised, I would suggest, we move forward with merging this PR.